data/team/src/commonMain/kotlin/com/adammcneilly/pocketleague/data/team/LiquipediaTeamRepository.kt

```diff
@@ -1,100 +1,100 @@
-package com.adammcneilly.pocketleague.data.team
-
-import com.adammcneilly.pocketleague.core.models.Team
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.flow
-import kotlinx.coroutines.withContext
-import org.jsoup.Jsoup
-import org.jsoup.nodes.Element
-
-/**
- * A custom implementation of [TeamRepository] that will send and request data
- * from the Liquipedia Rocket League website.
- */
-class LiquipediaTeamRepository : TeamRepository {
```

Comment:
Removed because jsoup wasn't working with iOS target enabled. Not a big deal yet, might need to fix again in the future.